### DISCUSSION TITLE
npx prisma init gives issue

### DISCUSSION THREAD

**Discussion by Ram-Poptani** [prisma/prisma#22318] [Q&A]
### Question

Whenever I am trying to generate prisma files it is giving me unable to get local issuer certificate 
![image](https://github.com/prisma/prisma/assets/58910170/53b0aa45-ed2a-4967-98f5-4be2daf5d764)

I think prisma is not even downloaded 
Even while I am trying to get prisma version
I am getting this
Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/79fb5193cf0a8fdbef536e4b4a159cad677ab1b9/windows/query_engine.dll.node.sha256 failed, reason: unable to get local issuer certificate

### Information about Prisma Schema, Client Queries and Environment (optional)

- OS: WIndows 10
- Database: PostgreSQL
- Node.js version: v20.5.0

**Comment by ludralph** [comment]
Hi @Ram-Poptani 👋 

Thank you for raising this question.

Can you please check that you can download this binary by clicking on the link below?

https://binaries.prisma.sh/all_commits/79fb5193cf0a8fdbef536e4b4a159cad677ab1b9/windows/query_engine.dll.node.sha256 

Also i do think the error has something to do with your local environment. Can you take a look at this [blog](https://cheapsslweb.com/blog/fixing-unable-to-get-issuer-cert-locally-error)? Let me know if the suggestions there help in resolving this issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
